integration-cli/docker_api_resize_test.go

```diff
@@ -42,7 +43,5 @@ func (s *DockerSuite) TestResizeApiResponseWhenContainerNotStarted(c *check.C) {
 	c.Assert(status, check.Equals, http.StatusInternalServerError)
 	c.Assert(err, check.IsNil)
 
-	if !strings.Contains(string(body), "Cannot resize container") && !strings.Contains(string(body), cleanedContainerID) {
-		c.Fatalf("resize should fail with message 'Cannot resize container' but instead received %s", string(body))
-	}
+	c.Assert(string(body), checker.Contains, cleanedContainerID, check.Commentf("resize should fail with message 'Cannot resize container'"))
```

Comment:
is this `!strings.Contains(string(body), "Cannot resize container")` not needed here anymore?